frontend/src/helpers/get-columns-helper/get-columns.helper.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  PermissionsGroupCreationDto,
+  UsersGetAllItemResponseDto,
+  UsersGroupCreationDto,
+} from 'common/types/types';
+import { Column } from 'react-table';
+
+const getColumns = (): Column<UsersGetAllItemResponseDto>[] => {
+  return [
+    {
+      Header: 'Id',
+      accessor: 'id',
+    },
+    {
+      Header: 'Email',
+      accessor: 'email',
+    },
+  ];
+};
+
+const getGroupCreationUserColumns = (): Column<UsersGroupCreationDto>[] => {
+  return [
+    {
+      Header: '',
+      accessor: 'checkbox',
```

Comment:
Use custom cell here instead of passing checkbox through data

Reply:
Not resolved. I tried doing it as you adviced, through headers & custom cell, but it didn't work out for me. Maybe I was trying in wrong way.